concepts/05 Widgets/Toolbar/05 Specify Item Type.md

```diff
@@ -123,6 +174,104 @@ Items that contain a widget should have the [widget](/api-reference/_hidden/dxTo
         // ...
     })
 
+##### Vue
+
+    <!--tab: App.vue-->
+    <template>
+        <DxToolbar>
+            <DxItem
+                widget="dxButton"
+                :options="buttonOptions"
+                location="before">
+            </DxItem>
+            <DxItem
+                widget="dxSelectBox"
+                :options="selectBoxOptions"
+                location="after">
+            </DxItem>
+        </DxToolbar>
+    </template>
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxToolbar, { DxItem } from 'devextreme-vue/toolbar';
+    import DxSelectBox from 'devextreme-vue/select-box';
```

Comment:
Don't you need to import the Button too?

Reply:
It is works without import the Button